base/loading.jl

```diff
@@ -508,7 +508,8 @@ function stale_cachefile(modpath, cachefile)
             return true # cache file was compiled from a different path
         end
         for (f,ftime) in files
-            if mtime(f) != ftime
+            # Issue #13606: compensate for Docker images rounding mtimes
+            if mtime(f) ∉ (ftime, floor(ftime))
```

Comment:
Talking about perf here is way over-optimizing, but on readability I found my brain reading this as mathematical set notation, rather than a 2-elem tuple.